airflow/www/static/js/dag/StatusBox.tsx

```diff
@@ -50,8 +60,15 @@ export const StatusWithNotes = ({
       background={getStatusBackgroundColor(color, !!containsNotes)}
       borderRadius="2px"
       borderWidth={state ? 0 : 1}
+      display="flex"
+      alignItems="center"
+      justifyContent="center"
+      lineHeight="10px"
+      fontSize="8px"
       {...rest}
-    />
+    >
+      {displayRetryIcon(tryNumber)}
```

Comment:
```suggestion
      {!!tryNumber && tryNumber > 1 && <MdRefresh color="white" />}
```

Let's just call the component instead of a function calling the component.

Reply:
I've made this change 👍